syntax = "proto3";

package deployment_plan;

service DeploymentPlanService {
    rpc GetDeploymentPlans(DeploymentPlanRequest) returns (stream DeploymentPlanResponse);
}

message DeploymentPlanRequest {
    string token = 1;
}

message DeploymentPlanResponse {
    string plan = 1;
    string instance_type = 2;
}